components/blitz/resources/omero/api/IAdmin.ice

```diff
@@ -18,52 +18,428 @@ module omero {
+                /**
+                 * Looks up an {@link omero.model.ExperimenterGroup} and all
+                 * contained {@link omero.model.Experimenter} users by name.
+                 *
+                 * @param groupName Name of the ExperimenterGroup
+                 * @return an ExperimenterGroup. Never null.
+                 * @throws ApiUsageException if groupName does not exist.
+                 */
                 idempotent omero::model::ExperimenterGroup lookupGroup(string name) throws ServerError ;
+
+                /**
+                 * Looks up all {@link omero.model.ExperimenterGroup} groups
+                 * present and all related
+                 * {@link omero.model.Experimenter} experimenters. The
+                 * experimenter's groups are also loaded.
+                 *
+                 * @return all Groups. Never null.
+                 */
                 idempotent ExperimenterGroupList lookupGroups() throws ServerError;
+                /**
+                 * Fetches all {@link omero.model.Experimenter} users
+                 * contained in this group. The returned users will have all
+                 * fields filled in and all collections unloaded.
+                 *
+                 * @param groupId id of the ExperimenterGroup
+                 * @return non-null array of all
+                 * {@link omero.model.Experimenter} users in this group.
+                 */
                 idempotent ExperimenterList containedExperimenters(long groupId) throws ServerError;
+
+                /**
+                 * Fetches all {@link omero.model.ExperimenterGroup} groups of
+                 * which the given user is a member. The returned groups will
+                 * have all fields filled in and all collections unloaded.
+                 *
+                 * @param experimenterId id of the Experimenter. Not null.
+                 * @return non-null array of all
+                 * {@link omero.model.ExperimenterGroup} groups for this user.
+                 */
                 idempotent ExperimenterGroupList containedGroups(long experimenterId) throws ServerError;
+
+                /**
+                 * Retrieves the default {@link omero.model.ExperimenterGroup}
+                 * group for the given user id.
+                 *
+                 * @param experimenterId of the Experimenter. Not null.
+                 * @return non-null {@link omero.model.ExperimenterGroup}. If
+                 *         no default group is found, an exception will be
+                 *         thrown.
+                 */
                 idempotent omero::model::ExperimenterGroup getDefaultGroup(long experimenterId) throws ServerError;
+
+                /**
+                 * Looks up {@link omero.model.Experimenter} experimenters who
+                 * use LDAP authentication  (has set dn on password table).
+                 *
+                 * @return Experimenter. Never null.
```

Comment:
There are quite a few cases where the documented param/return does not match the method param/return in various ways. Here the documented return type is an Experimenter but a string is returned - experimenter name? Should those be fixed here in this PR?